vendor/tektelic/T000489x-codec.yaml

```
@@ -0,0 +1,22 @@
+uplinkDecoder:
+  fileName: decoder_home_sensor.js
+  examples:
+    - description: periodic uplink
+      input:
+        fPort: 1
+        bytes: [03, 67, 00, 0A, 04, 68, 28]
```

Format as hex;
```suggestion
        bytes: [0x03, 0x67, 0x00, 0x0A, 0x04, 0x68, 0x28]
```

Note that `28` is not `0x28`